jablib/src/main/antlr/org/jabref/logic/journals/ltwa/Ltwa.g4

```diff
@@ -59,7 +59,14 @@ HYPHEN: '-';
 
 SYMBOLS: [.,;!?&+=*#%@$] | '\'';
 
-fragment LETTER: 'A'..'Z' | 'À'..'Ö' | 'ø'..'ÿ' | '\u0100'..'\u017F' | '\u4E00'..'\u9FFF';
+fragment LETTER:
+    [A-Z]
+  | [a-z]
+  | '\u00C0'..'\u00D6'   // À–Ö
+  | '\u00D8'..'\u00F6'   // Ø–ö
+  | '\u00F8'..'\u00FF'   // ø–ÿ
+  | '\u0100'..'\u017F'
```

Comment:
can you add a comment for the other unicode chars as welll? .e.g https://www.compart.com/de/unicode/U+0100